template/src/components/ui/Icon.tsx

```diff
@@ -0,0 +1,130 @@
+import { styled } from "@mui/material";
+import * as React from "react";
+
+type IIconNames = "close";
+
+const icons: { [k in IIconNames]: { path: React.ReactNode; fileName: string } } = {
+    close: {
+        fileName: "ic_close.svg",
+        path: (
+            // ic_close.svg
```

Comment:
Remove comment about file name because we anyway made it required to provide it via `fileName`.

Reply:
[remove useless icon name comment](https://github.com/allaboutapps/cra-template-aaa-typescript/pull/23/commits/b04bfd8e6f0818e59ed6614da5ed254d44a6a4e7)